### DISCUSSION TITLE
Welcome to GCD Discussions!

### DISCUSSION THREAD

**Discussion by joewheaton** [Riverscapes/gcd#405] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Post questions 
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by flaviamarini99** [comment]
Hi,
I've generated a DEM of Difference (DoD) using the GCD tool and reviewed the results in the Change Detection Results section. I noticed something that seems inconsistent and would like clarification:

In the statistics table, particularly under Vertical Averages, I see erosion values reaching approximately -4 meters.

However, when viewing the DoD raster layer with the default red/blue symbology, the legend only displays a range from < -2.5 m (erosion) to > +2.5 m (deposition).

I'm concerned that changes exceeding ±2.5 meters might not be properly visualized on the map.

Additionally, the histograms for area and volume show long tails reaching up to ±100 meters, but I don't see such extreme values visually represented in the raster.

Could you please clarify:

Are values greater than ±2.5 m actually present in the raster but just not visible due to the default legend settings?

Is there a recommended way to visualize the full range of elevation changes in the raster map?

Could those extreme values (e.g., ±100 m) be due to data artifacts or alignment errors between the input DEMs?

Thanks in advance!

**Reply by philipbaileynar** [reply]
Yes the default symbology is fixed at +- 2.5 m. You can use the built-in GIS symbology tools to adjust this level.

In a future version, we should probably consider making this 2.5 value a software setting that can be adjusted by the user.

**Reply by flaviamarini99** [reply]
Hi,

Thank you again for your helpful reply!

I noticed that you said the default range of the symbology is set at ±2.5 meters. However, in my latest analysis, the legend in the DoD raster layer goes from -2.20 to +2.20 meters, not ±2.5 and I have a net difference in thickness in the area of interest id -5.34 m

so where does the software get the 2.20 value from?

thanks

**Reply by philipbaileynar** [reply]
Here's the code where its scaling to +- 2.5 m

https://github.com/Riverscapes/gcd/blob/0b8d65f274b20d56a4ad92b27aa8a08183271e25/GCDAddIn/RasterSymbolization.cs#L612-L613
